src/cli/cmds/new.ts

```diff
@@ -452,10 +458,16 @@ async function initProject(
     );
     logging.debug("system using git version ", gitversion);
     if (gitversion && semver.gte(gitversion, "2.28.0")) {
-      git("init -b main");
+      const defaultGitInitBranch =
+        execOrUndefined("git config init.defaultBranch", {
+          cwd: baseDir,
+        })?.trim() ||
+        args.defaultReleaseBranch ||
```

Comment:
```suggestion
        args.defaultReleaseBranch ||
```

Would you be okay to leave this out? I don't really want to mix project settings with cli settings.
The way to get `defaultReleaseBranch` involved is imo to update the default for it to also respect the git setting.

Reply:
yes, sure